src/service/poc_mobile.proto

```diff
@@ -278,6 +278,33 @@ message verified_subscriber_location_ingest_report_v1 {
   uint64 timestamp = 3;
 }
 
+message subscriber_referral_eligibility_req_v1 {
+  // Id of the subsciber which will get the referral rewards
+  bytes subscriber_id = 1;
+  // Shares rewardsed to the subscriber
+  uint64 shares = 2;
+  // Pubkey of the carrier for which the subscriber is attached
+  bytes carrier_pub_key = 3;
+  // Signed payload of the request
+  bytes signature = 4;
+}
+
+message subscriber_referral_eligibility_resp_v1 { string id = 1; }
+
+enum subscriber_referral_eligibility_status {
+  referral_eligibility_status_valid = 0;
+  referral_eligibility_status_invalid_subscriber_id = 1;
+  referral_eligibility_status_invalid_carrier_key = 2;
+}
+
+message subscriber_referral_eligibility_ingest_report_v1 {
+  subscriber_referral_eligibility_req_v1 report = 1;
+  // Status of the report
+  subscriber_referral_eligibility_status status = 2;
```

Comment:
Not sure why the ingest report has a status.  I believe you need to create a `verified_subscriber_referral_eligibility_ingest_report_v1` and the status should be there.